sbin/cubeit

```diff
@@ -257,6 +257,7 @@ done
 
 trap_cmd='trap_handler $?'
 trap "${trap_cmd}" EXIT
+trap "clean_up \"Action cancelled by user.\"" SIGINT
```

Comment:
It would be better to funnel both of the traps through the same routine. i.e. put this clean up in the existing trap_handler (Which can be overridden by installers if needed).